Lombiq.HelpfulLibraries/Docs/Navigation.md

```diff
@@ -0,0 +1,5 @@
+# Navigation Documentation
+
+## PortalNavigationProviderBase

```

Comment:
I commented it to the base theme, but I'll leave it here too:

> I think the word Portal is too specific. With this widget, you are providing the ability to render navigation anywhere. Maybe NavigationWidgetFilter would be more appropriate.

Reply:
I'm providing for the navigation providers with the name "portal". I can make some abstract base classes if we anticipate the need for other navigation types, but I thought we can do that when needed. Am I missing some consideration?

Reply:
nvm, I've created a base class with the overridable NavigationName

Reply:
My issue is with the class name and basically the widget name. It can be any navigation and the name "portal" in the class name indicates something more specific. The navigation name itself is another story, but that can be generic, too. The word "portal" is coming from SNOW in the first place.

Reply:
Okay, now I'm confused. wtf is a "portal" anyway? I'm only used to _thinking with portals_ :)
Wikipedia says "a specially designed website that brings information from diverse sources, like emails, online forums and search engines, together in a uniform way". Is that applicable to SNOW?